greenwood.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     { property: 'og:title', content: 'Greenwood' },
     { property: 'og:type', content: 'website' },
     { property: 'og:url', content: 'https://www.greenwoodjs.io' },
-    { property: 'og:image', content: 'https://s3.amazonaws.com/hosted.greenwoodjs.io/greenwood-logo.png' },
+    { property: 'og:image', content: 'https://www.greenwoodjs.io/assets/greenwood-logo-300w.png' },
```

Comment:
`300w` seems the right size for a card. I presume, but we have a few sizes to pick from if we need an alternate
https://github.com/ProjectEvergreen/greenwood/tree/master/www/assets